examples/openshift-gitops/subscription.yaml

```diff
@@ -1,3 +1,19 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-gitops-operator
+
+---
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: openshift-gitops-operator
+  namespace: openshift-gitops-operator
+spec:
+  upgradeStrategy: Default
+
+---
```

Comment:
Added this couple of resources that were missing from the openshift-gitops install example.